exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/Serializer.java

```diff
@@ -200,6 +217,24 @@ public void serializeRepeatedFixed64(ProtoFieldInfo field, long[] values) throws
     writeEndRepeatedPrimitive();
   }
 
+  /** Serializes a {@code repeated uint64} field. */
+  public void serializeRepeatedUInt64(ProtoFieldInfo field, List<Long> values) throws IOException {
+    if (values.isEmpty()) {
+      return;
+    }
+
+    int payloadSize = 0;
+    for (long v : values) {
```

Comment:
We know it's a `PrimitiveLongList` in practice so let's use `PrimitiveLongList.toArray` first. I guess I would make this method accept `long[]` and the caller do that conversion.

Same for sizing

Reply:
Ah, nice I wasn't aware you changed the `MetricData` to use the `PrimitiveLongList`. I've changed the bucket marshaller to use `PrimitiveLongList.toArray(counts)` when calling the serializer 👍